assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_grading_submission.vm

```diff
@@ -240,12 +240,32 @@
         #supplementItems($!allowViewModelAnswer $!allowReadAssignmentNoteItem $!allowViewAllPurposeItem $!assignmentModelAnswerItem $!assignmentNoteItem $!assignmentAllPurposeItem)
 
         #if ($!hasAssociatedRubric)
-        <ul id="rubricTabs" class="nav nav-tabs">
-            <li class="active"><a href="#submition">$tlang.getString("gen.subm")</a></li>
-            <li><a href="#rubric">$tlang.getString("grading_rubric")</a></li>
+        <ul id="rubricTabs" class="nav nav-tabs mt-2">
+            <li class="nav-item">
+                <button type="button"
+                        role="tab"
+                        class="nav-link active"
+                        data-bs-target="#submission"
+                        data-bs-toggle="tab"
+                        aria-controls="submission"
+                        aria-selected="true">
+                    $tlang.getString("gen.subm")
```

Comment:
Can we do something about "gen.subm"? Can we change the property to gen.submit or something that is readable and doesn't feel like a spelling mistake?

Reply:
We could update that - it would have to be updated across all the languages for that properties file though. And, you'll find that they are all named in a similar way and you'd want to change all of them :)